packages/api/database/migrations/1595452149359_technology_costs_schema.js

```diff
@@ -18,7 +18,8 @@ class TechnologyCostsSchema extends Schema {
 				.unique()
 				.unsigned()
 				.references('id')
-				.inTable('technologies');
+				.inTable('technologies')
+				.onDelete('cascade');
```

Comment:
Acho que seria bacana aproveitar e fazer o update também.
```suggestion
				.onUpdate('cascade')
				.onDelete('cascade');
```